app/views/users/shared/_otp_delivery_preference_selection.html.erb

```diff
@@ -1,41 +1,9 @@
-<%
-  form_name = form_obj.class.name.underscore.to_s
-  form_name_label = form_name + "[otp_delivery_preference]"
-  form_name_tag_sms = form_name + "_otp_delivery_preference_sms"
-  form_name_tag_voice = form_name + "_otp_delivery_preference_voice"
-%>
-
-<div class="mb3">
-  <fieldset class="m0 p0 border-none">
-    <legend class="mb1 h4 serif bold">
-      <%= t('two_factor_authentication.otp_delivery_preference.title') %>
-    </legend>
-    <p class="mt0 mb2" id="otp_delivery_preference_instruction">
-      <%= t('two_factor_authentication.otp_delivery_preference.instruction') %>
-    </p>
-    <label
-      class="btn-border col-12 sm-col-5 sm-mr2 mb2 sm-mb0"
-      for= <%= form_name_tag_sms %>
-    >
-      <div class="radio">
-        <%= radio_button_tag form_name_label,
-          :sms, form_obj.delivery_preference_sms?,
-          class: :otp_delivery_preference_sms %>
-        <span class="indicator"></span>
-        <%= t('two_factor_authentication.otp_delivery_preference.sms') %>
-      </div>
-    </label>
-    <label
-      class="btn-border col-12 sm-col-5 mb0"
-      for=<%= form_name_tag_voice %>
-    >
-      <div class="radio">
-        <%= radio_button_tag form_name_label,
-            :voice, form_obj.delivery_preference_voice?,
-            class: :otp_delivery_preference_voice %>
-        <span class="indicator"></span>
-        <%= t('two_factor_authentication.otp_delivery_preference.voice') %>
-      </div>
-    </label>
-  </fieldset>
-</div>
+<%= render 'shared/alert', type: 'warning', class: 'margin-bottom-2' do %>
+  <p>
+    We are having trouble making phone calls to confirm phone numbers.
```

Comment:
Looks like there are translations for this copy but this is still hardcoded

Reply:
Lol, flawless execution on my part. Willfix